docs/source/starthere/tutorials.rst

```diff
@@ -107,14 +107,8 @@ To run a tutorial:
      - Multi-lingual ASR
      - `Multi-lingual ASR <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/Multilang_ASR.ipynb>`_
    * - ASR
-     - Hybrid ASR-TTS Models
-     - `Hybrid ASR-TTS Models <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/ASR_TTS_Tutorial.ipynb>`_
-   * - ASR
-     - ASR Confidence Estimation
-     - `ASR Confidence Estimation <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/ASR_Confidence_Estimation.ipynb>`_
-   * - ASR
-     - Confidence-based Ensembles
-     - `Confidence-based Ensembles <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/Confidence_Ensembles.ipynb>`_
+     - Hybrid ASR-TTS Models Tutorial
```

Comment:
Hmm  why is ASR Confidence Estimation being removed ? Plus Confidence based Ensembles. @Kipok is this intentional?

Reply:
Not sure about the Hybrid models, but confidence tutorials should not be removed. I think there was a PR that mistakenly removed them and maybe it was only fixed in main and not in the release branch?

Reply:
It might need to be reverted I could have resolved the conflict incorrectly

Reply:
@ericharper  seems this removal was a mistake. Can you add back these deleted rows and keep the added items?